content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -1,65 +1,67 @@
-=====================
-Add optional products
-=====================
+=================
+Optional products
+=================
 
-The use of optional products is a marketing strategy for cross-selling products along with a core
-product. The aim is to offer useful and related products to your customers. For instance, if a
-customer wants to buy a car, he has the choice to order an automatic opening trunk and massaging
-seats, or not to order such high-quality products and simply buy his car.
+The use of optional products is a marketing strategy that involves the cross-selling of products
+along with a core product. The aim is to offer useful and related products to customers, which may
+result in an increased sale.
 
-Add optional products to your quotations
-========================================
+For instance, if a customer wants to buy a car, they have the choice to order massaging seats, as
+well, or ignore the offer and simply buy the car. Presenting the choice to purchase optional
+products enhances the customer experience.
+
+Optional products on quotations
+===============================
 
-With Odoo Sales, it is possible to add or modify optional products directly on quotations
-(under the **Optional Products** tab, as you can see below).
+With the Odoo *Sales* application, it is possible to add or modify optional products directly on
+quotations (located under the :guilabel:`Optional Products` tab on a quotation form, in the *Sales*
+application).
 
-.. image:: optional_products/optional_products_2.png
+.. image:: optional_products/optional-products-2.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to add optional products to your quotations on Odoo Sales
+   :alt: How to add optional products to your quotations on Odoo Sales.
 
 .. note::
-   By clicking on the **Customer Preview** button, you will be able to see what your customers
-   will have as possibilities after opening a quotation from their received email.
+   Clicking the :guilabel:`Customer Preview` button, reveals a preview of the quotation customers
+   receive via email, along with the optional products they have a chance of adding to their order.
 
-   .. image:: optional_products/optional_products_3.png
+   .. image:: optional_products/optional-products-3.png
       :align: center
-      :class: img-thumbnail
-      :alt: Preview your quotations on Odoo Sales
+      :alt: Preview your quotations on Odoo Sales.
+
+In practice, customers are able to add different optional products to an order by clicking the
+:guilabel:`Shopping Cart icon`, located to the right of the optional product line.
```

Comment:
If referring to an icon, place only the icon descriptive text in the backticks of the guilabel, leaving the word "icon" outside and only capitalize proper nouns:
```suggestion
:guilabel:`shopping cart` icon, located to the right of the optional product line.
```
Just as an FYI - if you chose to use an icon inside the guilabel it would look like this:
```suggestion
:guilabel:`🛒 (shopping cart)` icon, located to the right of the optional product line.
```